CodingTracker/Viewer.cs

```diff
@@ -0,0 +1,83 @@
+﻿using ConsoleTableExt;
+using System.Globalization;
+
+namespace CodingTracker
+{
+    internal static class Viewer
+    {
+        public static void DisplayPromptForTime(string verb)
+        {
+            Console.WriteLine($"\nWhen did you {verb} coding? Please answer below in the following format: dd-MM-yy HH-mm-ss");
```

Comment:
I like the way you used one Console.WriteLine for both messages. Very clever. I would remove the need to add seconds though. It would be better UX and seconds are irrelevant in this business case